### PR TITLE
Ivan view follower

### PR DESCRIPTION
**Pull request by ivanseit2001**
Added unfriending

### CODE REVIEW

**Comment by Jdvdb** [comment]
I'm getting an operational error because 'no such table: quickcomm_follow_request' when trying to go to the profile page. I've tried makemigrations and migrate but there's nothing new if I try to do this. Is there a step I'm missing?

File: images/Image_created_with_a_mobile_phone.png
Comment (Jdvdb) [inline images/Image_created_with_a_mobile_phone.png:1]: I don't think we want to commit images into the main repo

**Comment by Jdvdb** [comment]
The friend request system seems buggy. For example, after accepting a request from a user, it replaced that friend request with a friend request from myself. If I accept that, then it breaks the website and saves some kind of incomplete following relationship. I somehow was able to follow someone twice as a user:
Screenshot 2023-03-22 at 5 29 58 PM

Other than that it seems to be good! Just double-check the way you handle logic when someone creates a friend request and when someone accepts it.

File: quickcomm/models.py [hunk 1]
```diff
@@ -86,14 +97,26 @@ def save(self, *args, **kwargs):
...
 
+    
+
     def is_bidirectional(self):
         """Returns true if the follow is bidirectional."""
         return self.following.is_following(self.follower)
 
     def __str__(self):
         return f"{self.follower.__str__()} follows {self.following.__str__()}"
+class follow_request(models.Model):
```
Comment (micheal-chow) [inline quickcomm/models.py:116]: Should probably stick the current class naming conventions, such that this would be `FollowRequest`

**Pull request merged**